doc/getting-started/installation-instructions/slurm.md

```diff
@@ -42,19 +43,19 @@ of nodes which is too large you can have one of the following:
    `Resources` (waiting for resources to become available) or `Priority`
    (queued behind a higher priority job).
 
-If you are stuck in situation 2 you must kill the openquake job and the
+If you are stuck in situation 2 you must kill the
 SLURM job with the command `scancel JOBID` (JOBID is listed by the
 command `$ squeue -u $USER`). If you are stuck in situation 3 for a long
-time it can be better to kill the jobs (both openquake and SLURM) and
-then relaunch the calculations, this time asking for fewer nodes.
+time it can be better to kill the job and
+then relaunch the calculation, this time asking for fewer nodes.
 
 ## Running out of quota
 
 The engine will store the calculation files in `shared_dir`
 and some auxiliary files in `custom_dir`; both directories are
 mandatory and must be specified in the configuration file. The
-`shared_dir` is meant to point to the work area of the cluster
-and the `custom_tmp` to the scratch area of the cluster.
+`shared_dir` should be locateded in the work area of the cluster
+and the `custom_tmp` in the scratch area of the cluster.
 
```

Comment:
Need to mention that those dir need to be shared from management and calcs nodes?

Reply:
A cluster admin knows what is a "work" or a "scratch" area.